doc/user/content/sql/system-catalog/mz_internal.md

```diff
@@ -68,6 +68,7 @@ granted the [`mz_monitor` role](/manage/access-control/manage-roles#builtin-role
 | `finished_at`              | [`timestamp with time zone`] | The wall-clock time at which the statement finished executing.                                                                                                                                                                                                                |
 | `finished_status`          | [`text`]                     | The final status of the statement (e.g., `success`, `canceled`, `error`, or `aborted`). `aborted` means that Materialize exited before the statement finished executing.                                                                                                    |
 | `error_message`            | [`text`]                     | The error message, if the statement failed.                                                                                                                                                                                                                                   |
+| `error_message_redacted`   | [`text`]                     | The redacted error message, if the statement failed.                                                                                                                                                                                                                          |
```

Comment:
One quick naming nit: `mz_internal.mz_notices` does the naming the other way around (i.e., `redacted_error_message`), as do all the `create_sql`/`redacted_create_sql` columns in `mz_catalog`.